assets/js/webinar.js

```diff
@@ -0,0 +1,48 @@
+/**
+ *
+ * @param {string} date The string value of the date to localize.
+ */
+function localizeDateTime(date) {
+    var localeDate = new Date(date.getAttribute("data-datetime"));
+    var options = { timeZoneName: "short" };
+    return localeDate.toLocaleString(undefined, options);
+}
+
+$(function() {
```

Comment:
It's fine to keep this, but going forward, now that @cnunciato has hooked up Stencil.js builds in this repo, for all new JavaScript written, we should really consider leveraging that and phasing out our usage of JQuery. See https://github.com/pulumi/docs/tree/master/components. This way we can use modern language features in TypeScript, real components, etc.

Reply:
👍  Sounds good.